phy/cluster/manual/session.py

```diff
@@ -346,24 +322,36 @@ def redo(self):
     # Event callbacks
     # -------------------------------------------------------------------------
 
-    def on_open(self):
-        """Update the session after new data has been loaded."""
-        self._global_history = GlobalHistory(process_ups=_process_ups)
-        # TODO: call this after the channel groups has changed.
-        # Update the Selector and Clustering instances using the Model.
-        spike_clusters = self.model.spike_clusters
-        self.clustering = Clustering(spike_clusters)
-        self.cluster_metadata = self.model.cluster_metadata
-        # TODO: n_spikes_max in a user parameter
-        self.selector = Selector(spike_clusters, n_spikes_max=100)
-
-        # Kwik store.
-        path = _ensure_disk_store_exists(self.model.name,
-                                         root_path=self._store_path,
-                                         )
+    def _load_default_settings(self):
+        """Load default settings for manual clustering."""
+        curdir = op.dirname(op.realpath(__file__))
+        # This is a namespace available in the config file.
+        file_namespace = {
+            'n_spikes': self.model.n_spikes,
+            'n_channels': self.model.n_channels,
+        }
+        settings.set_user(path=op.join(curdir, 'default_settings.py'),
```

Comment:
from the line I was thinking set_user was loading the default_settings file!

from phy.utils import USER_SETTINGS, INTERNAL_SETTINGS ?
from phy.utils import user_settings, internal_settings ?

USER_SETTINGS.set()